extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -88,7 +88,9 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     try {
       const { raw } = await this.view.emailProvider.msgGet(msgId, 'raw', progress => this.setQuoteLoaderProgress(progress));
       this.setQuoteLoaderProgress('processing...');
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      if (typeof raw === 'undefined') {
+        return undefined;
+      }
```

Comment:
you should remove `!` from `raw!` below on line 94

Reply:
updated. thanks!